### PR TITLE
Allow fulltext search on template description

### PR DESCRIPTION
**Pull request by sliverc**
This only works for database engines which support it.
Currently mysql and postgresql.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -4,6 +4,8 @@ django-filter==2.0.0
 djangorestframework==3.9.0
 docx-mailmerge==0.4.0
 docxtpl==0.5.14
+mysqlclient==1.3.13
+psycopg2-binary==2.7.6.1
```
Comment (winged) [inline requirements.txt]: Duplicate requirement

File: document_merge_service/settings.py [hunk 2]
```diff
@@ -28,6 +28,24 @@ def default(default_dev=env.NOTSET, default_prod=env.NOTSET):
...
+        ),
+        "USER": env.str("DATABASE_USER", default=""),
+        "PASSWORD": env.str("DATABASE_PASSWORD", default=""),
+        "HOST": env.str("DATABASE_HOST", default=""),
+        "PORT": env.str("DATABASE_PORT", default=""),
+        "OPTIONS": env.dict("DATABASE_OPTIONS", default={}),
+    }
+}
+
+
```
Comment (winged) [inline document_merge_service/settings.py]: Why the move of the `DATABASES` block?

Comment (sliverc) [reply document_merge_service/settings.py]: I need to be able to access DATABASES['default']['ENGINE'] to be able to determine whether postgres needs to be added to `INSTALLED_APPS`.

File: requirements.txt (hunk 1 above)
Comment (sliverc) [reply requirements.txt]: ouch. That was still there from the cookiecutter. Removed.

**Pull request merged**